ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ImmutableDB/StateMachine.hs

```diff
@@ -457,8 +472,9 @@ generator m@Model {..} = do
   where
     -- Don't simulate an error during corruption, because we don't want an
     -- error to happen while we corrupt a file.
-    errorFor Corruption {} = False
-    errorFor _             = True
+    errorFor Corruption {}      = False
+    errorFor ReopenInThePast {} = False
```

Comment:
Make a ticket for this, because we're not checking that truncating blocks from the future behaves correctly in the presence of simulated errors.

Reply:
See #1567.